extensions/ql-vscode/CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 - Fix proper escaping of backslashes in SARIF message strings.
 - Allow setting `codeQL.runningQueries.numberOfThreads` and `codeQL.runningTests.numberOfThreads` to 0, (which is interpreted as 'use one thread per core on the machine').
 - Clear the problems view of all CodeQL query results when a database is removed.
+- Add the view DIL command on a query history item that opens the Datalog Intermediary Language text in an editor.
```

Comment:
```suggestion
- Add the `View DIL` command on a query history item. This opens a text editor containing the Datalog Intermediary Language representation of the compiled query.
```

Keeping the wording consistent with https://help.semmle.com/codeql/codeql-cli/commands/query-decompile.html.

Reply:
Made slight changes, but used your suggestion.